### PR TITLE
10-final snischuk

### PR DESCRIPTION
**Pull request by snischuk**
Previous PR: #425 

Deploy: https://snischuk.github.io/fe-course-basic-2023/

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @snischuk, if you like to close one pr, and create a new one with the same changes, for better traceability, you can add here link to the closed pr https://github.com/MastersAcademy/fe-course-basic-2023/pull/425

File: docs/snischuk/images/.DS_Store
Comment (vladyslav-yermolin-moc) [inline docs/snischuk/images/.DS_Store:1]: Please remove this file

File: docs/snischuk/scripts/pokemons.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+            ThumbnailAltText,
+            height,
+            weight,
+            number,
+            type,
+            weakness,
+        } = pokemon;
+
+        const typesButtons = type
+            .map((subtype) => `<button class="card-pokemon__btn card-pokemon__btn--${subtype.toLowerCase()}" type="button">${subtype.charAt(0).toUpperCase()}${subtype.toLowerCase().slice(1)}</button>`)
```
Comment (vladyslav-yermolin-moc) [inline docs/snischuk/scripts/pokemons.js:75]: I think this is not the buttons - they don't do anything on click. 
image

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: docs/snischuk/images/.DS_Store
Comment (snischuk) [reply docs/snischuk/images/.DS_Store:1]: Fixed

File: docs/snischuk/scripts/pokemons.js (hunk 1 above)
Comment (snischuk) [reply docs/snischuk/scripts/pokemons.js]: Fixed

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Comment by vladyslav-yermolin-moc** [comment]
Well done! 👏

**Review by vladyslav-yermolin-moc — APPROVED** [review]
ok, I'm merging it.
Thanks for your efforts, @snischuk

**Pull request merged**